debug-single-test.sh

```diff
@@ -4,10 +4,56 @@ SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
 source ${SCRIPT_DIR}/docker/dev.env
 
-TEST_CASE_NAME=$1
-echo "Testing ${TEST_CASE_NAME}"
+function print_usage() {
+  printf '\n%b\n\n' "Usage: ${0} [ -a APPLICATION_NAME ] [ -s SCHEMA_VERSION ]"
+  printf '%b\n' "Debugging a given unit/integration test from the given application using the given schema version"
```

Comment:
I think this wording is simpler and more to the point:
“Debug a unit/integration test in a module with the given schema version”

Also, I think the usage should include an explanation of the `-n` option.

Reply:
Good spot. Thanks, fixed both